README_zh_CN.md

```diff
@@ -0,0 +1,23 @@
+[English](README.md) | [简体中文](README_zh_CN.md)
+
+
+
+# Genshin Schedule
+
+Genshin Schedule 是一个用于帮助你规划《原神》中与时间相关的活动的网站.
+
+更多信息请访问: [genshin.chiya.dev](https://genshin.chiya.dev).
+
+![home](images/home.png)
+
+## 从源代码构建
+
+这个网站由两个子项目 [web](web) 和 [sync](sync) 组成
+
+- `web` 前端网页
+- `sync` 后端, 处理API请求
+
+详细构建过程请前往子项目查看
+
+You may use the provided Dockerfiles ([web](Dockerfile.web) and [sync](Dockerfile.sync)) to generate production images. 
```

Comment:
Why did you leave this sentence untranslated?

Reply:
Since I haven't used/understood Docker, I can't guarantee that it won't be translated wrong.

Reply:
I added the translation of this sentence after I learned something about it, but I added the original sentence because I didn't know whether completely correct.